json/Item_Weapon_TwinDagger.txt

```diff
@@ -1220,21 +1220,21 @@
 	{
 		"assign": "140319",
 		"jp_text": "深遠刃オブセシオン",
-		"tr_text": "Profound Obsessxion",
+		"tr_text": "Profound Obsexion",
 		"jp_explain": "終わりなき物語を辿る者の絶望により\n生み出された双小剣。重苦と破滅を\n滲ませた刃が、負の力を増大させる。",
 		"tr_explain": ""
 	},
 	{
 		"assign": "140320",
 		"jp_text": "星震刃オブセシオン",
-		"tr_text": "Celestial Obsessxion",
+		"tr_text": "Celestial Obsexion",
 		"jp_explain": "時を遡る者の世界を想う心によって\n覚醒した双小剣。全てに勝る強き想いが\n闇を掻き消し、刃に光を取り戻す。",
 		"tr_explain": ""
 	},
 	{
 		"assign": "140321",
 		"jp_text": "光跡刃オブセシオン",
-		"tr_text": "Lightstream Obsessxion",
+		"tr_text": "Lightstream Obsexion",
```

Comment:
was trying to avoid this

Reply:
I know it doesn't look fantastic, but as you had it it would have been pronounced
"Ob-sess-sheeon"
I don't think there's any way to write it that looks good in english, what with it being a double letter this time.

Reply:
That's what it's supposed to be though, not all of them are in english.